labman/gui/handlers/plate.py

```diff
@@ -81,7 +81,9 @@ def get(self):
                       if plate_type is not None else None)
         only_quantified = True if only_quantified == 'true' else False
 
-        rows_list = [[p['plate_id'], p['external_id'],
+        rows_list = [[p['plate_id'],
+                      p['external_id'],
+                      str(p['creation_timestamp']),
```

Comment:
Just to confirm, it's more desirable to have p['creation_timestamp'] retain any time-stamping methods and force a str() for perhaps the most used case, rather than convert it to a string value internally?

Reply:
Good point